modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -41,9 +41,9 @@ The inbound load balancer runs on each controller node configured in Runtime Fab
 
 .. Select a mode.
 
-*** *Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
+*** *Shared Mode* is the default setting if no dedicated internal load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
 
-*** *Dedicated Mode* is enabled only if one or more dedicated internal load balancer nodes is available in Runtime Fabric. Because dedicated internal load balancer nodes use all available resources for running the internal load balancer, you cannot specify the amount of CPU cores and memory. If you are setting up a Runtime Fabric with dedicated nodes, include the IP addresses of the dedicated load balancer nodes only when configuring the external load balancer.
+*** *Dedicated Mode* is enabled only if one or more dedicated internal load balancer nodes is available in Runtime Fabric. Because dedicated internal load balancer nodes use all available resources for running the internal load balancer, you cannot specify the amount of CPU cores and memory. If you are setting up a Runtime Fabric with dedicated internal load balancer nodes, include the IP addresses of the dedicated internal load balancer nodes only when configuring the external load balancer.
```

Comment:
```suggestion
*** *Dedicated Mode* is enabled only if one or more dedicated internal load balancer nodes are available in Runtime Fabric. Because dedicated internal load balancer nodes use all available resources for running the internal load balancer, you cannot specify the amount of CPU cores and memory. If you are setting up a Runtime Fabric with dedicated internal load balancer nodes, include the IP addresses of the dedicated internal load balancer nodes only when configuring the external load balancer.
```